Pipfile

```diff
@@ -27,12 +27,15 @@ requests = "==2.18.0"
 sorl-thumbnail = "==12.3"
 python-dateutil = "*"
 futures = "*"
+"bfb3076" = {file = "https://github.com/mtlpy/django-slackin/archive/make-it-work.zip"}
```

Comment:
wat ?

Reply:
Tu n'aimes pas ce petit côté *scrappy*? :trollface:

Reply:
Seriously, I want to see if this package is good enough for us. If it is, I will put a bit more effort into it (like publish it to pypi)

Reply:
hahaha on devrait au moins renomer en `bfg9000`